### ISSUE TITLE
Slow performace when using ORDER BY

### ISSUE THREAD

**Issue by Ryiski** [parseablehq/parseable#430]
I've noticed queries are running extremely slow when I try and `ORDER BY` on the logs

You can give it a test on Postman

Slow query:
```json
{
    "query": "SELECT * FROM frontend ORDER BY p_timestamp LIMIT 30 OFFSET 0",
    "startTime": "2022-03-01T00:00:00Z",
    "endTime": "2023-06-03T23:59:59.999Z"
}
```

Fast query:
```json
{
    "query": "SELECT * FROM frontend LIMIT 30 OFFSET 0",
    "startTime": "2022-03-01T00:00:00Z",
    "endTime": "2023-06-03T23:59:59.999Z"
}
```

**Comment by nitisht** [comment]
Thanks @Ryiski we’ll take a look.

**Comment by nitisht** [comment]
Will be fixed by #443
